docs/config.md

```diff
@@ -85,33 +93,25 @@ Obviously, you would only set one line, don't copy them all in!
 
 ## Diffing
```

Comment:
Since this section is only about editing diffs (not about displaying them), should we make the heading "Editing diffs"?

Reply:
Sounds good.